addons/osfstorage/models.py

```diff
@@ -560,3 +561,20 @@ def create_waterbutler_log(self, auth, action, metadata):
             auth=auth,
             params=params
         )
+
+    @property
+    def storage_usage(self):
+        storage_usage = cache.get('storage_usage')
+
+        if storage_usage:
+            return storage_usage
+
+        file_versions = OsfStorageFile.objects.filter(provider='osfstorage',
+                                                   target_object_id=self.owner_id,
+                                                   versions__isnull=False).values_list('versions',
+                                                                                       flat=True)
+        # An empty queryset will produce a None instead of 0
+        sum = FileVersion.objects.filter(id__in=file_versions).aggregate(sum=models.Sum('size'))['sum'] or 0
```

Comment:
The cache isn't being used--this query is being called no matter what, which defeats the purpose of caching.

You'll need to

1. Try to get the cached storage usage value for the node
2. If it doesn't exist (or it's expired), then run the query and cache the value for 60 seconds.
3. If it does exist, return that value.

Reply:
Also, a nit: `sum` is a reserved word. How about `value` or `total`?

Reply:
I'm confused by this because I'm pretty sure I am actually this is working as intended (aside from the cacheing on a per-node basis). If the value has been cached the method exits early [here](https://github.com/CenterForOpenScience/osf.io/pull/8414/files/00c56995d72fad3e1cf9cab0b7754ad5966ebd0d#diff-511d556f4513d9e41850aa51126eceabR570). otherwise the query runs. I even have to [bust the cache](https://github.com/CenterForOpenScience/osf.io/pull/8414/files/00c56995d72fad3e1cf9cab0b7754ad5966ebd0d#diff-df93ad330ce69a1ef2564e15f3ac0bd6R636) for the unit test to work